src/super_gradients/training/utils/optimizers/all_optimizers.py

```diff
@@ -0,0 +1,11 @@
+import warnings
+
+from super_gradients.common.registry.registry import OPTIMIZERS as CURRENT_VERSION_OPTIMIZERS
+
+OPTIMIZERS = CURRENT_VERSION_OPTIMIZERS
+
+warnings.warn(
```

Comment:
why are we not using the same concept of `make_deprecated`?

Reply:
Because the entirely_optimizers.py file was removed, it made more sense to me to warn any user that would try and access this file.